purchases/src/test/java/com/revenuecat/purchases/common/offerings/OfferingsFactoryTest.kt

```diff
@@ -43,17 +45,18 @@ class OfferingsFactoryTest {
 
     private lateinit var billing: BillingAbstract
     private lateinit var offeringParser: OfferingParser
-
+    private lateinit var dispatcher: Dispatcher
     private lateinit var offeringsFactory: OfferingsFactory
 
     @Before
     fun setUp() {
         billing = mockk()
         offeringParser = GoogleOfferingParser()
-
+        dispatcher = SyncDispatcher()
         offeringsFactory = OfferingsFactory(
-            billing = billing,
-            offeringParser = offeringParser
+            billing,
+            offeringParser,
+            dispatcher
         )
     }
```

Comment:
I am not entirely sure what else to test here. The issue was due to `billing.queryProductDetailsAsync(` returning in a different thread, but we can't really test that 🤔